.env.template

```diff
@@ -0,0 +1,14 @@
+DJANGO_ALLOWED_HOSTS="0.0.0.0,127.0.0.1"
```

Comment:
Can you please remove Airtable environmental Variables?, we no longer use Airtable in the project.

Reply:
Oh, is that so? Very well, then.